Engine/src/Engine/DeltaTime.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+namespace Engine
+{
+	//Container for the DeltaTime used in game.
+	class DeltaTime
+	{
+	public:
+		DeltaTime(double dt = 0)
+		: m_Time(dt)
+		{
+		}
+
+		operator double() const { return m_Time; }
+
```

Comment:
If I understand correctly, these return deltatime in milliseconds or seconds, yes?